blenderproc/python/types/EntityUtility.py

```diff
@@ -61,12 +70,29 @@ def get_location(self, frame: int = None) -> np.ndarray:
     def get_rotation(self, frame: int = None) -> np.ndarray:
         """ Returns the rotation of the entity in euler angles.
 
+        :param frame: The frame number at which the value should be returned. If None is given, the current frame number is used.
+        :return: The rotation at the specified frame.
+        """
+        warnings.warn("This function will be deprecated. Use get_rotation_euler() instead.")
+        return self.get_rotation_euler(frame)
+
+    def get_rotation_euler(self, frame: int = None) -> np.ndarray:
+        """ Returns the rotation of the entity in euler angles.
+
         :param frame: The frame number at which the value should be returned. If None is given, the current frame number is used.
         :return: The rotation at the specified frame.
         """
         with KeyFrame(frame):
             return np.array(self.blender_obj.rotation_euler)
+        
+    def get_rotation_mat(self, frame: int = None) -> np.ndarray:
+        """ Gets the rotation matrix of the entity.
 
+        :param frame: The frame number which the value should be set to. If None is given, the current frame number is used.
+        :return: The 3x3 local2world rotation matrix.
+        """
+        return np.array(Euler(self.get_rotation(frame)).to_matrix())
```

Comment:
Should use the non-deprecated function
```suggestion
        return np.array(Euler(self.get_rotation_euler(frame)).to_matrix())
```

Reply:
fixed sorry